lib/puppet/info_service/task_information_service.rb

```diff
@@ -0,0 +1,25 @@
+class Puppet::InfoService::TaskInformationService
+
```

Comment:
should this `require 'puppet/module'` or can we safely assume that will already be loaded?